docs/navbar.js

```diff
@@ -66,3 +70,17 @@ function loadNavbarWithPicker(
 function getUrlUnderDocs(pathFragment) {
   return import.meta.url.replace(/navbar\.js$/, pathFragment);
 }
+
+/**
+ * Gets the url for the help section relevant to the current page
+ * @return {string}
+ */
+function getHelpUrl() {
+  if (window.location.pathname === '/') {
```

Comment:
Are these paths ok when running under a subdirectory?

Reply:
Updated the logic to handle this case